ui/component/dateTime/view.jsx

```diff
@@ -1,71 +1,42 @@
 // @flow
-import React from 'react';
+import { getTimeAgoStr } from 'util/time';
+import classnames from 'classnames';
 import moment from 'moment';
+import React from 'react';
 
 const DEFAULT_MIN_UPDATE_DELTA_MS = 60 * 1000;
 
+type State = {
+  lastRenderTime: Date,
+};
+
 type Props = {
-  date?: any,
-  timeAgo?: boolean,
-  formatOptions?: {},
-  show?: string,
   clock24h?: boolean,
+  date?: any,
   minUpdateDeltaMs?: number,
-};
-
-type State = {
-  lastRenderTime: Date,
+  type?: string,
+  showOnHover?: boolean,
+  timeAgo?: boolean,
+  setDebouncedDate?: (any) => void,
```

Comment:
In trying to avoid two-direction data flow, I wonder if we can just use the React key to force an update to the date component.
- In `LivestreamComment`:
    - `const [dateLastUpdate, setDateLastUpdate] = useState(0)`
    - If `timestamp` still requires the 30s update, install the timer in `useEffect`
        - Timer will `setDateLastUpdate(Date.now())`.  The value doesn't matter, just needs to be unique to spark a redraw.
    - `<DateTime date={timestamp * 1000} ... key={dateLastUpdate} />`

Optional:
Taking this one step further up, we could also avoid potentially having 50 timers for 50 comments by having `LivestreamComments` (the parent) do the work with just 1 timer.  Just brainstorming at this point, but maybe something like (minimal code):
- If any comment requires the 30s update, install timer.
- `const [dateLastUpdate, setDateLastUpdate] = useState(0)`
- Calling `setDateLastUpdate` in the timer should spark a render, which should cause `LivestreamComment` to render.  i.e. just refresh the whole pane as if a new comment was received.